src/cascadia/TerminalApp/TerminalPage.idl

```diff
@@ -14,10 +19,12 @@ namespace TerminalApp
         String ApplicationDisplayName { get; };
         String ApplicationVersion { get; };
 
+        // We cannot use the default XAML APIs because of a bug, see GH#5224.
```

Comment:
we also want to make sure that there's only one application-global dialog visible at a time 😉 that's more the purpose of the presenter